### PR TITLE
ADEN-5876 Enable Kikimora for adblock traffic 

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/provider/gpt/helper.js [hunk 1]
```diff
@@ -119,6 +121,9 @@ define('ext.wikia.adEngine.provider.gpt.helper', [
 				slotTargetingData.src = 'premium';
 			} else if (adShouldBeRecovered) {
 				slotTargetingData.src = 'rec';
+			} else if (instartLogic && instartLogic.isEnabled() && instartLogic.isBlocking()) {
```
Comment (kvas-damian) [inline extensions/wikia/AdEngine/js/provider/gpt/helper.js]: Are you sure we don't want set recovery targeting on premium pages?

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [googleTag.js#L11](https://github.com/Wikia/app/blob/901459cd50245e1acf0fb06e13d7d02799485288/extensions/wikia/AdEngine/js/provider/gpt/googleTag.js#L11): Remove the unused function parameter "instartLogic". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by kvas-damian — APPROVED** [review]

File: extensions/wikia/AdEngine/js/provider/gpt/helper.js [hunk 2]
```diff
@@ -121,6 +123,11 @@ define('ext.wikia.adEngine.provider.gpt.helper', [
 				slotTargetingData.src = 'rec';
 			}
 
+			if (instartLogic && instartLogic.isEnabled() && instartLogic.isBlocking()) {
```
Comment (dianafa) [inline extensions/wikia/AdEngine/js/provider/gpt/helper.js:128]: this is not needed anymore - IL adds those params automatically

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [googleTag.js#L11](https://github.com/Wikia/app/blob/52d7e9e8c52719ba99b10dee7ef56d6d48a78c6a/extensions/wikia/AdEngine/js/provider/gpt/googleTag.js#L11): Remove the unused function parameter "instartLogic". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**